samples/samples-js/AddProduct/index.js

```diff
@@ -0,0 +1,16 @@
+module.exports = async function (context, req) {
+    context.log('JavaScript HTTP trigger function processed a request.');
+
+    const testProduct = {
```

Comment:
These samples should be behaving identically - so this one should match the C# one which is pulling from the body

Reply:
yeah but when am testing locally not from tests, I wanted it to return the data when I invoke it from browser so added it. Would it be okay to leave it as is?